src/components/member/tabs/quote-tab/quote-activation.tsx

```diff
@@ -37,43 +30,37 @@ export const QuoteActivation: React.FC<{
   onSubmitted = noopFunction,
   onWipChange = noopFunction,
 }) => {
-  const [activationDate, setActivationDate] = React.useState<Date | null>(
-    new Date(),
-  )
-  const [terminationDate, setTerminationDate] = React.useState<Date | null>(
-    null,
-  )
   const [useGap, setUseGap] = React.useState(false)
+  const [contracts] = useContracts(memberId)
+  const contract = getContract(contracts, quote)
+  const [activeFrom, setActiveFrom] = React.useState<Date | null>(new Date())
```

Comment:
Not sure if we want this to be nullable
```suggestion
  const [activeFrom, setActiveFrom] = React.useState<Date>(new Date())
```